lib/customer.rb

```diff
@@ -0,0 +1,45 @@
+require_relative "dishes"
+
+class Customer
```

Comment:
could have split this class into multiple classes as it has multiple responsibilities